chain/mocks_test.go

```diff
@@ -0,0 +1,180 @@
+package chain
+
+import (
+	"container/list"
+	"errors"
+	"testing"
+
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/btcsuite/btcd/btcutil/gcs"
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightninglabs/neutrino"
+	"github.com/lightninglabs/neutrino/banman"
+	"github.com/lightninglabs/neutrino/headerfs"
+)
+
+var (
+	ErrNotImplemented                      = errors.New("not implemented")
+	_                 NeutrinoRescanner    = (*mockRescanner)(nil)
+	_                 NeutrinoChainService = (*mockChainService)(nil)
+	testBestBlock                          = &headerfs.BlockStamp{
+		Height: 42,
+	}
+)
+
+func newMockNeutrinoClient(t *testing.T,
+	opts ...func(*mockRescanner)) *NeutrinoClient {
+	t.Helper()
+	var (
+		chainParams  = &chaincfg.Params{}
```

Comment:
A package global variable called `chainParams` is already defined, can just use that one.